src/Simulation/Algorithms/COE/RV2COE.hpp

```
@@ -0,0 +1,196 @@
+
+            if (n.y < 0.0)
+                omega = TWOPI - omega;
+        }
+
+
+        // Orbit type
+        Physics::OrbitGeometry orbitGeom = ELLIPTICAL;
+        Physics::OrbitInclination orbitIncl = INCLINED;
+
+            // Orbit geometry
+        if (e_mag < R_EPSILON)                                          // e = 0
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)